docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+
+services:
+  app:
+    container_name: kukkuu-admin
+    build:
+      context: .
+      target: ${DOCKER_TARGET:-development}
+    volumes:
+      - '.:/app'
+      - '/app/node_modules'
+    ports:
+      - '3000:3000'
```

Comment:
By the way: It is likely that a developer wants to run both kukkuu-ui and kukkuu-admin on the same computer. Maybe this one should run on 3001 or something?

Reply:
Yeah good point, I will change that.